odata-protocol/11.4 Data Modification.md

```diff
@@ -178,13 +178,22 @@ values beyond those specified in the metadata SHOULD NOT be sent in the
 request body. The service MUST fail if unable to persist all property
 values specified in the request.
 
-Properties computed by the service (annotated with the term
-[`Core.Computed`](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Core.V1.md#Computed),
-see [OData-VocCore](#ODataVocCore)) and properties that are tied to
-properties of the principal entity by a referential constraint, can be
-omitted and MUST be ignored if included in the request.
-
-Properties with a defined default value, nullable properties, and
+Non-insertable properties, as well as dependent properties that are tied to
+non-key properties of the principal entity, SHOULD be omitted from the request body.
```

Comment:
I think I have understood it now. "Tied to" means there exists a referential constraint.
```suggestion
Non-insertable properties, as well as dependent properties with a referential constraint whose principal property lies outside the key, SHOULD be omitted from the request body.
```